schema/tables/pwd_policy.yml

```diff
@@ -0,0 +1,28 @@
+name: pwd_policy
+platforms:
+  - darwin
+description: Password Policiy (e.g max failed password attempts).
+columns:
+  - name: maxFailedAttempts
+    type: integer
+    required: <=5
+    description: |
+      The account lockout threshold specifies the amount of times a user can enter an incorrect password before a lockout will occur. Ensure that a lockout threshold is part of the password policy on the computer.
+  - name: expiresEveryNDays
+    type: integer
+    required: <=365
+    description: |
+      How many days for a new password to expire.
+  - name: daysToExpiration
+    type: integer
+    required: <=365
+    description: |
+      How many days are left for the expiration of the current password.
+  - name: historyDepth
+    type: integer
+    required: >=15
```

Comment:
The website tests are currently failing because of these columns' `required` values. 
A column's `required` value must be a boolean representing whether or not the column is required to query the table.

Reply:
Thanks Eric.
Fixed.